docs/examples.md

```diff
@@ -553,6 +553,34 @@ rspec_task:
     Which will create four shards that will theoretically **run tests 4x faster** by equaly splitting all tests between 
     these four shards.
 
+### RSpec and RuboCop Annotations
+
+Cirrus CI natively supports [RSpec](https://rspec.info/) and [RuboCop](https://rubocop.org/) machine-parsable JSON reports.
+
+To get Behaviour-Driven Development test annotations, simply generate a `rspec` artifact from your linter task:
```

Comment:
Please use en-US spelling, and this sentence can be improved a bit:
```suggestion
To get behavior-driven test annotations, simply generate a `rspec` artifact from your lint task:
```

Reply:
Thanks!